### ISSUE TITLE
init_image support

### ISSUE THREAD

**Issue by dribnet** [eps696/aphantasia#32]
Do any of the drawing modules have support for initialising from an image? I looked but didn't see any inverse FFT code currently in the codebase. If not this might be an interesting feature to consider adding for any models that would easily support such an operation.

**Comment by eps696** [comment]
all three main methods can start from an image - [fft](https://github.com/eps696/aphantasia/blob/a431bb81e3de41d896a7dda1196317b874c974d5/clip_fft.py#L278), [dwt](https://github.com/eps696/aphantasia/blob/a431bb81e3de41d896a7dda1196317b874c974d5/clip_fft.py#L152), [rgb](https://github.com/eps696/aphantasia/blob/a431bb81e3de41d896a7dda1196317b874c974d5/clip_fft.py#L175).
this feature is very hacky [and kinda broken] though, in line with my understanding of the functions from the lucent lib. the colors usually become seriously distorted on the load; so i would definitely appreciate any hints regarding fixing this.

**Comment by dribnet** [comment]
Thanks - I've found this in the code via the "resume" argument. Agreed the current implementation is a bit rough around the edges, but a useful baseline we might be able to improve over time.

**Comment by htoyryla** [comment]
I was looking for a way to initialize fft from an image, for a setup that uses FFT for image generation and VGG19 style transfer and CLIP for guiding the iterations. My FFT implementation was borrowed from here, and I had to rework it (largely by trial and error) to get it to start from a good reproduction of the init image.

You'll find my code here https://github.com/htoyryla/fftstylediver

It will probably not work correctly on the latest pytorch versions (have not tested it and I have changed the FFT dimension thing). I have also done some scaling things differently, and dropped to_valid_rgb from the chain.
